operator/pkg/manifest/installer.go

```diff
@@ -495,39 +495,56 @@ func buildComponentApplyOutput(stdout string, stderr string, objects object.K8sO
 	}
 }
 
+func istioCustomResources(group string) bool {
+	switch group {
+	case "config.istio.io",
+		"rbac.istio.io",
+		"security.istio.io",
+		"authentication.istio.io",
+		"networking.istio.io":
+		return true
+	}
+	return false
+}
+
 // DefaultObjectOrder is default sorting function used to sort k8s objects.
 func DefaultObjectOrder() func(o *object.K8sObject) int {
```

Comment:
https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#organizing-resource-configurations

kubectkl will be created in the order they appear in the file.

Reply:
Wow, I am very surprised by this. Well, I am surprised they recommend it rather, since even if its in order its still eventually consistent so I am surprised it has a real impact. Thanks for the reference